test/mpi/f08/pt2pt/irsendf08.f90

```diff
@@ -25,20 +24,22 @@ program isend
          call test_pair_irsend( comm, errs )
          call mtestFreeComm( comm )
       enddo
-!
+
```

Comment:
did you mean to remove the `!` at the beginning of this line?

Reply:
This was an accident